contrib/vstudio/readme.txt

```diff
@@ -43,6 +43,10 @@ Build instructions for Visual Studio 2015 (32 bits or 64 bits)
 - Decompress current zlib, including all contrib/* files
 - Open contrib\vstudio\vc14\zlibvc.sln with Microsoft Visual C++ 2015
 
+Build instructions for Visual Studio 2017 (32 bits or 64 bits)
+--------------------------------------------------------------
+- Decompress current zlib, including all contrib/* files
+- Open contrib\vstudio\vc15\zlibvc.sln with Microsoft Visual C++ 2017

```

Comment:
if `vc` here stands for VisualC++, `15` might technically be wrong?  It's confusing but it looks like `2017` is v14.x ( https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B )?

Reply:
That's the value for the MSVC macro, but it does say Visual Studio itself is version 15:

`MSVC++ 14.1  _MSC_VER == 1910 (Visual Studio 2017 version 15.0)`

Reply:
Oh hang on, I see what you mean, you're looking at `vc`. I can change it to 14.14 if you feel strongly about it.

Reply:
I looked to see if anyone tried to do similar changes on the main repo and found this
https://github.com/madler/zlib/pull/243

They had the same debate and seemed like they were leaning towards `vc15` since `vc14` doesn't make a lot of sense anyways so I'm good with 15 :+1: